client/rest-high-level/src/main/java/org/elasticsearch/client/indices/DataStream.java

```diff
@@ -90,6 +92,10 @@ public boolean isSystem() {
         return system;
     }
 
+    public boolean isAllowCustomRouting() {
```

Comment:
Minor wording change:
```suggestion
    public boolean allowsCustomRouting() {
```

Reply:
done